jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/impl/Bpmn2UnmarshallingTest.java

```diff
@@ -1533,6 +1533,14 @@ public void testMITaskProperties() throws Exception {
     }
 
     @Test
+    public void testNoMIOnEmbeddedSubprocess() throws Exception {
+        Definitions definitions = loader.loadProcessFromJson("simpleEmbeddedSubprocess.json");
+        Process process = getRootProcess(definitions);
+        SubProcess subprocess = (SubProcess) getFlowElement(process.getFlowElements(), "abc");
+        assertNull(subprocess.getLoopCharacteristics().getId());
```

Comment:
I think the assert should looks like:
assertNull(subprocess.getLoopCharacteristics());